Makefile

```diff
@@ -14,15 +14,15 @@
 
 TARGET = eventrouter
 GOTARGET = github.com/openshift/$(TARGET)
-LOCAL_IMAGE_TAG=openshift/logging-eventrouter
-IMAGE_REPOSITORY_NAME=quay.io/openshift/logging-eventrouter:latest
+IMAGE_REPOSITORY_NAME ?= quay.io/openshift/logging-eventrouter:latest
```

Comment:
please change this to "openshift-logging"

Reply:
done